doc/update-copyright

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+## ---------------------------------------------------------------------
+##
+## Copyright (C)  by the authors of the ASPECT code.
```

Comment:
I noticed that too, and could have sworn I fixed it! The reason is that the script ignores its self because it is only associated with one commit, whose message includes something about updating copyrights :) Definitely should be fixed somehow.